### PR TITLE
Updated background image and font style

### PR DESCRIPTION
**Pull request by Mohit049**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
     </nav>
 
     <main>
-      <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
+      <h1 class="welcome-title" style=" font-size:50px;font-family:algerian,verdana; color:navy;"> Welcome to Real Dev Squad </h1>
```
Comment (ankushdharkar) [inline index.html:23]: Let's move this into the main css file please

File: index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap" rel="stylesheet">
 </head>
 
-<body>
+<body style="background-image:url(https://theartmad.com/wp-content/uploads/2015/09/Best-Light-Website-Backgrounds-5.jpg);>
```
Comment (ankushdharkar) [inline index.html:14]: Let's keep the css in it's css files please. No inline styling is preferrable

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
@Mohit049 Thanks for the PR. Some small changes are needed before we can approve and merge this. Thanks

**Comment by swarajpure** [comment]
@Mohit049 are you going to make the changes?

**Pull request closed without merging**